src/Toggle/MysqlActivatedToggleGateway.php

```diff
@@ -43,20 +62,21 @@ public function __construct( Connection $connection, IsToggleActive $toggleCheck
      */
     public function getAllMyActivatedToggles( $userIdentifier )
```

Comment:
I agree. But after a discussion with Viktor this is what we agreed on, namely to keep this method name and remove the argument. I also has the same opinion, that lab library implementations shouldn't take current user and group as a global knowledge because it's weird to require "current" user and group objects when there's not even a login session term existing in this context. The constructor isn't part of the interface and the interface should always make it clear if something requires a user ID and group ID. @viktorcb thought otherwise and I couldn't argue with the fact that probably these classes won't be ever called for multiple users/groups.